SCClassLibrary/Common/Collections/windows/extString_windows.sc

```diff
@@ -1,6 +1,6 @@
 + String {
 	runInTerminal {
-		"String:runInTerminal - sorry this method has not yet been implemented for windows.".error;
+		format("start \"SuperCollider runInTerminal\" cmd.exe /k %", this.quote).unixCmd;
```

Comment:
i don't think this does quote escaping right, but i just realized we can avoid that whole problem by using Array's unixCmd method, which makes escaping unnecessary since the args are passed directly to the new process as given.

Reply:
are you sure? I tested this and a) this works fine for path like `"c:\\program files (x86)\\something.exe".runInTerminal`, b) using `["start cmd.exe \k", this].unixCmd` doesn't work without quoting the string.

Reply:
i was thinking of this: https://stackoverflow.com/a/15262019. spaces are not the concern, quote characters are.

you're right about array.unixCmd. i missed that the actually array passed to execvp on Linux is prepended with "/bin/sh -c", and on Windows i didn't read too closely but i think the strings are just directly concatenated...

let's not assume that openOS is correct; it might be worth looking at what other languages do if you have the time.

in any case, as long as you've tested this reasonably well i'm okay with it. but, i think we should acknowledge that there's a lot of inconsistency among all the SuperCollider functions that invoke the system shell, that's technical debt for the future i guess.